src/ideal/ideal.jl

```diff
@@ -646,6 +646,20 @@ function reduce(a::T, b::T) where T <: SPolyUnion
    return R(ptr)
 end
 
+@doc Markdown.doc"""
+    division(I::sideal{S}, G::sideal{S}) where S <: SPolyUnion
+
+Computes a division with remainder.
+returns a list T,Rest,U where matrix(I)*matrix(U)=matrix(G)*matrix(T)+matrix(Rest)
```

Comment:
```suggestion
Returns a tuple `T,Rem,U` where `matrix(I)*matrix(U)=matrix(G)*matrix(T)+matrix(Rem)`.
```

Reply:
This seems underspecified -- returning three zero matrices would satisfy this definition, wouldn't it?

Reply:
I used the documentation for `lift` to hopefully provide a better doc string now.